src/coreComponents/constitutive/fluid/multifluid/compositional/models/CriticalVolume.hpp

```diff
@@ -2,10 +2,10 @@
  * ------------------------------------------------------------------------------------------------------------
  * SPDX-License-Identifier: LGPL-2.1-only
  *
- * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
- * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
- * Copyright (c) 2018-2020 TotalEnergies
- * Copyright (c) 2019-     GEOSX Contributors
+ * Copyright (c) 2018-2024 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2024 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2024 TotalEnergies
+ * Copyright (c) 2019-     GEOS/GEOSX Contributors
```

Comment:
just curious - why this header is different across the files?
there is Chevron in the file below and above but not here?

Reply:
Probably because someone created a file in a PR using the old header, then when I changed the header, they didn't update their copy, then it was merged into develop, then that header was copied.....or I just messed up.

Reply:
@paveltomin This should be fixed. If you happen to see anything else while you are randomly reviewing the 2000 files let me know 😁